src/greatwall/resources/knowledge/fractal.py

```diff
@@ -38,6 +39,7 @@ def __init__(
         self.height: Optional[int] = height
         self.escape_radius: Optional[int] = escape_radius
         self.max_iters: Optional[int] = max_iters
+        self.computed_values = computed_values
```

Comment:
I will actually prefer renaming attribute to something like that `_fractal_iteration_cache`, what do you think?

Reply:
Yes I like it. Also since it is a dictionary it could be used for more values that we want to cache if needed. Good naming.